scripts/install-nix-from-closure.sh

```diff
@@ -246,6 +246,21 @@ if [ -z "$NIX_INSTALLER_NO_MODIFY_PROFILE" ]; then
             break
         fi
     done
+
+    # For Fish
+    # Make the shell source nix.fish during login.
+    p=$HOME/.nix-profile/etc/profile.d/nix.fish
+    for i in .config/fish/config.fish; do
```

Comment:
We don't need the for if the file can only be in one location.

Reply:
I agree. I wanted to reuse the same pattern for code consistency. I will simplify this.